solidity/contracts/domains/interfaces/IZetoPrivate.sol

```diff
@@ -2,9 +2,12 @@
 pragma solidity ^0.8.20;
 
 /**
- * @title IZetoPrivate
+ * @title IZetoPrivate (Deprecated)
```

Comment:
no need to have this to complicate things as we haven't had any official releases.